test/features/transaction/transaction_tile_test.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:didpay/features/transaction/transaction.dart';
+import 'package:didpay/features/transaction/transaction_details_page.dart';
+import 'package:didpay/features/transaction/transaction_notifier.dart';
+import 'package:didpay/features/transaction/transaction_tile.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/mocks.dart';
+import '../../helpers/test_data.dart';
+import '../../helpers/widget_helpers.dart';
+
+void main() {
+  group('TransactionTile', () {
+    final pfi = TestData.getPfi('did:dht:pfiDid');
+    const exchangeId = 'rfq_01ha835rhefwmagsknrrhvaa0k';
```

Comment:
is `exchangeId` being used for anything other than initializing `transactionTileTestWidget`? if not, this variable probably isn't necessary

Reply:
It's only used to initialize the `TransactionTile` widget. 

I'll remove it the extra variable.